src/components/shared/Head.astro

```diff
@@ -28,10 +28,10 @@ const socialImageURL = new URL(Astro.props.image ?? '/meta-image.jpg', Astro.url
   <link rel="apple-touch-icon" sizes="180x180" href="/favicon/apple-touch-icon.png" />
   <link rel="icon" type="image/png" sizes="32x32" href="/favicon/favicon-32x32.png" />
   <link rel="icon" type="image/png" sizes="16x16" href="/favicon/favicon-16x16.png" />
-  <link rel="manifest" href="/site.webmanifest" />
-  <link rel="mask-icon" href="/favicon/safari-pinned-tab.svg" color="#ee7711" />
+  <link rel="manifest" href="/favicon/site.webmanifest" />
+  <link rel="mask-icon" href="/favicon/safari-pinned-tab.svg" color="#5bbad5" />
   <meta name="msapplication-TileColor" content="#da532c" />
-  <meta name="theme-color" content="#fff" />
+  <meta name="theme-color" content="#ffffff" />
```

Comment:
Theme-color bych nastavil na hlavni pozadi co tam mame `#fffbd6`, at se to hezky zobrazuje na mobilu